pkg/agent/controller/agent_test.go

```diff
@@ -182,6 +182,11 @@ var _ = Describe("ServiceImport syncing", func() {
 					Protocol: corev1.ProtocolTCP,
 					Port:     123,
 				},
+				{
```

Comment:
AFAICT, the test doesn’t actually verify that the ports have been copied, does it?

Reply:
It does. Tom already added that check as part of UT
https://github.com/submariner-io/lighthouse/blob/66088a5575c1b3359fc4a5329570686fd7389546/pkg/agent/controller/agent_test.go#L541